editor/src/clj/editor/gl/shader.clj

```diff
@@ -333,6 +335,11 @@ This must be submitted to the driver for compilation before you can use it. See
   ;; value to set.
   )
 
+(defmethod set-uniforms-at-index (class (float-array []))
+  [^GL2 gl progn loc count vals]
+  (let [fb (FloatBuffer/wrap vals)]
+    (.glUniform4fv gl loc count fb)))
```

Comment:
Currently, the flattening of the buffer into a float array is done from the outside.
We can support other types later on.